ivy/functional/backends/jax/elementwise.py

```diff
@@ -12,7 +12,13 @@
 from . import backend_version
 
 
-def abs(x: Union[float, JaxArray], /, *, out: Optional[JaxArray] = None) -> JaxArray:
+def abs(
+    x: Union[float, JaxArray],
+    /,
+    *,
+    out: Optional[JaxArray] = None,
```

Comment:
following the convention we've followed through the codebase, `where` should come before `out`. This applies to all backend implementations and the array and container instance and static methods.